search/replicate-xapian-database

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Usage: $0 <LOCAL_DB_PATH> <PORT> <REMOTEHOST> <REMOTEUSER> <REMOTEPATH> <SSH_IDENTITY_FILE> [<REMOTE_LOCKFILE_PATH> [<REMOTE_RUN_WITH_LOCKFILE_BINARY_PATH>]]
+
+# Exit on error
+set -e
+
+#
+cd "$(dirname "$0")"/..
+
+# Enable job control and batch mode
+set -mb
+
+# Args
+LOCAL_DB_PATH="$1"
+PORT="$2"
+REMOTEHOST="$3"
+REMOTEUSER="$4"
+REMOTEPATH="$5"
+IDENTITY_FILE="$6"
+
+if [ "x$8" != "x" ]; then
```

Comment:
I think a couple of these var numbers are incorrect, due to the removal of the rails env one.